### PR TITLE
Read-only configuration facets not returned

### PR DESCRIPTION
**Pull request by zryska**
This adds information about handling of read-only facets on the root configuration controller level.

### CODE REVIEW

File: shared-content/configuration/system-and-adapter.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ uid: SystemAndAdapterConfiguration
 
 # System and adapter
 
-You can configure the system component and adapter component together using a single file.
+You can retrieve the configuration of, and configure the system component and adapter components together using a single file.
```
Comment (aveva-jhwang) [inline shared-content/configuration/system-and-adapter.md:7]: Maybe we can change the wording to: "You can retrieve and modify the configuration of the system component and adapter components together using a single file." ? If "modify" isn't the best word choice, we can adjust it.

File: shared-content/configuration/system-and-adapter.md [hunk 2]
```diff
@@ -35,8 +35,25 @@ Complete the following steps to configure system and adapter. Use the `PUT` meth
 
 If the operation fails due to errors in the configuration, the count of the error and suitable error messages are returned in the result.
 
+## Retrieve system and adapter configuration
+
+The entire application can be retrieved by issuing the following cURL command:
```
Comment (aveva-jhwang) [inline shared-content/configuration/system-and-adapter.md:40]: Do we want to specify "The entire application configuration"?

File: shared-content/configuration/system-and-adapter.md (hunk 1 above)
Comment (zryska) [reply shared-content/configuration/system-and-adapter.md]: That's perfect! I changed the word modify to set. I am happy to adjust further if needed.

File: shared-content/configuration/system-and-adapter.md (hunk 2 above)
Comment (zryska) [reply shared-content/configuration/system-and-adapter.md]: Great catch! Fixed. Thank you, Jiyeon!

Comment (rmuolic) [inline shared-content/configuration/system-and-adapter.md]:
```suggestion
The entire application can be retrieved by entering the following cURL command:
```

**Review by rmuolic — CHANGES_REQUESTED** [review]
Minor suggestion noted in my review. I've also changed the base branch to v1.6.

File: shared-content/configuration/system-and-adapter.md (hunk 2 above)
Comment (zryska) [reply shared-content/configuration/system-and-adapter.md]: I like the suggestion! Please feel free to add it in.

**Review by rmuolic — APPROVED** [review]

**Pull request merged**